Makefile

```diff
@@ -300,10 +300,13 @@ manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and Cust
 dependencies-manifests: export AUTHORINO_OPERATOR_GITREF := $(AUTHORINO_OPERATOR_GITREF)
 dependencies-manifests: export LIMITADOR_OPERATOR_GITREF := $(LIMITADOR_OPERATOR_GITREF)
 dependencies-manifests: export DNS_OPERATOR_GITREF := $(DNS_OPERATOR_GITREF)
+dependencies-manifests: export IMAGE_TAG := $(IMAGE_TAG)
 dependencies-manifests: ## Update kuadrant dependencies manifests.
 	$(call patch-config,config/dependencies/authorino/kustomization.template.yaml,config/dependencies/authorino/kustomization.yaml)
 	$(call patch-config,config/dependencies/limitador/kustomization.template.yaml,config/dependencies/limitador/kustomization.yaml)
 	$(call patch-config,config/dependencies/dns/kustomization.template.yaml,config/dependencies/dns/kustomization.yaml)
+	$(call patch-config,config/install/namespaced-dns-operator/dns-operator/kustomization.template.yaml,config/install/namespaced-dns-operator/dns-operator/kustomization.yaml)
+	$(call patch-config,config/install/standard/kuadrant-version.template.yaml,config/install/standard/kuadrant-version.yaml)
```

Comment:
@maleck13 I added your existing kuadrant install to this task. When we create a release, this will update the kuadrant version to the correct value based on the "VERSION" variable currently set. In the case of main, that will be "latest", but for a semantic version it will be the expected tag e.g. "v1.0.0-rc6"